src/main/resources/resources/logisim/strings/soc/soc_zh.properties

```diff
@@ -321,40 +321,40 @@ SocPioRisingEdge = 上升沿
 #
 # pio/PioMenu.java
 #
-ErrorCreatingHeaderAndOrCFile = 无法创建头文件和/或c文件！
-ExportC = 导出C模板
-PioMenuBidirFunctionsRemark = /*使用以下功能可以设置或读取双向寄存器。\n*1表示尊重。位是输出，0表示它是输入\n*注意：只使用较低的%s位。\n*/
+ErrorCreatingHeaderAndOrCFile = 无法创建头文件和/或 C 文件！
+ExportC = 导出 C 模板
+PioMenuBidirFunctionsRemark = /*使用以下功能可以设置或读取双向寄存器。\n*1 表示尊重。位是输出，0 表示它是输入\n*注意：只使用较低的 %s 位。\n*/
 PioMenuCaptureAll = 独立于参数“VALUE”的值清除捕获寄存器中的所有位
-PioMenuCaptureAny = 至少从0更改为1或从1更改为0
-PioMenuCaptureBit = 清除捕获寄存器中参数“VALUE”中为1的位
-PioMenuCaptureFalling = 至少一个从1到0的更改
-PioMenuCaptureRising = 至少一个从0到1的更改
-PioMenuEdgeCaptureRemark = /*使用以下功能可以清除或读出边沿捕获寄存器\n*如果读出边沿寄存器，\n*如果读出边沿寄存器，则相应的输入\n*具有HAT%s，\n*否则相应的输入没有更改。\n*\n*写入此寄存器将为%s。\n*注意：仅使用较低的%s位。\n*/
-PioMenuInputDataFunctionRemark = /*使用以下函数可以\n*读出PIO的输入值。\n*此函数返回各自的值\n*注意：只有较低的%s位有效\n*/
+PioMenuCaptureAny = 至少从 0 更改为 1 或从 1 更改为 0
+PioMenuCaptureBit = 清除捕获寄存器中参数“VALUE”中为 1 的位
+PioMenuCaptureFalling = 至少一个从 1 到 0 的更改
+PioMenuCaptureRising = 至少一个从 0 到 1 的更改
+PioMenuEdgeCaptureRemark = /*使用以下功能可以清除或读出边沿捕获寄存器\n*如果读出边沿寄存器，\n*如果读出边沿寄存器，则相应的输入\n*具有 HAT%s，\n*否则相应的输入没有更改。\n*\n*写入此寄存器将为 %s。\n*注意：仅使用较低的 %s 位。\n*/
+PioMenuInputDataFunctionRemark = /*使用以下函数可以\n*读出 PIO 的输入值。\n*此函数返回各自的值\n*注意：只有较低的 %s 位有效\n*/
 PioMenuIrqEdge = 捕获寄存器中的相应位
 PioMenuIrqLevel = 相应的输入
-PioMenuMaskFunctionsRemark = /*使用以下函数可以设置或读出中断屏蔽寄存器。\n*特定位置上的1表示如果%s等于1，\n*将生成中断。\n*注意：仅使用较低的%s位。\n*/
-PioMenuOutClearRemark = /*使用以下函数，参数“value”中为1的位将强制\n*相应的输出为0\n*注意：仅使用较低的%s位。\n*/
-PioMenuOutputDataFunctionRemark = /*使用以下函数可以设置\n*PIO的输出值。\n*此函数将新的输出值作为参数“value”\n*注意：只有提供的参数“value”的低%s位才有效\n*/
-PioMenuOutSetRemark = /*使用以下函数，参数“value”中为1的位将强制\n*相应的输出为1\n*注意：仅使用较低的%s位。\n*/
-SelectDirectoryToStoreC = 设置存储C模板的目录
-SuccesCreatingHeaderAndCFile = 已成功创建文件：\n%s\n和\n%s
+PioMenuMaskFunctionsRemark = /*使用以下函数可以设置或读出中断屏蔽寄存器。\n*特定位置上的 1 表示如果 %s 等于 1，\n*将生成中断。\n*注意：仅使用较低的 %s 位。\n*/
+PioMenuOutClearRemark = /*使用以下函数，参数“value”中为 1 的位将强制\n*相应的输出为 0\n*注意：仅使用较低的 %s 位。\n*/
+PioMenuOutputDataFunctionRemark = /*使用以下函数可以设置\n*PIO 的输出值。\n*此函数将新的输出值作为参数“value”\n*注意：只有提供的参数“value”的低 %s 位才有效\n*/
+PioMenuOutSetRemark = /*使用以下函数，参数“value”中为 1 的位将强制\n*相应的输出为 1\n*注意：仅使用较低的 %s 位。\n*/
+SelectDirectoryToStoreC = 设置存储 C 模板的目录
+SuccesCreatingHeaderAndCFile = 已成功创建文件：\n%s\n 和\n%s
 #
 # pio/SocPio.java
 #
-SocPioBidirPinx = 双目录位%s
+SocPioBidirPinx = 双目录位 %s
 SocPioBidirs = 双向销
 SocPioComponent = 并行输入/输出扩展器
-SocPioInputPinx = 输入位%s
+SocPioInputPinx = 输入位 %s
 SocPioInputs = 输入引脚
-SocPioIrqOutput = IRQ输出
-SocPioOutputPinx = 输出位%s
+SocPioIrqOutput = IRQ 输出
+SocPioOutputPinx = 输出位 %s
 SocPioOutputs = 输出引脚
 SocPioResetInput = 重置输入
 #
 # rv32im/Rv32im_riscv.java
 #
-Rv32imComponent = RISC V IM模拟器
+Rv32imComponent = RISC V IM 模拟器
```

Comment:
Use "RISC-V" instead of "RISC V" systematically.

Reply:
Probably DeepL issue. Fixed.